### PR TITLE
refactor(api): api_nodes spec use dashboard_swagger

### PR DESCRIPTION
**Pull request by JimMoen**

### CODE REVIEW

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 1]
```diff
@@ -17,121 +17,177 @@
...
+            }
+     };
+schema("/nodes/:node/metrics") ->
+    #{ 'operationId' => node_metrics
+     , get =>
+           #{ description => <<"Get node metrics">>
+            , parameters => [ref(node)]
+            , responses =>
+                  #{ 200 => mk( ref(?NODE_METRICS_MODULE, node_metrics)
+                              , #{dsec => <<"Get node metrics successfully">>})
```
Comment (HJianBo) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:86]:
```suggestion
                              , #{desc => <<"Get node metrics successfully">>})
```

**Review by HJianBo — APPROVED** [review]

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 2]
```diff
@@ -17,121 +17,177 @@
...
+      , mk(binary()
+          , #{ in => path
+             , description => <<"Node name">>
+             , required => true
+             , example => <<"emqx@127.0.0.1">>
+             })
+     }
+    ];
+fields(node_info) ->
+    [ { node
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:118]: change all node from string to  atom, then we don't need to binary_to_atom/2
```erlang
node(get, #{bindings := #{node_name := NodeName}}) ->
    get_node(binary_to_atom(NodeName, utf8)).
```

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 3]
```diff
@@ -17,121 +17,177 @@
...
+    , { load1
+      , mk(string(), #{ desc => <<"CPU average load in 1 minute">>
+                      , example => "2.66"})}
+    , { load5
+      , mk(string(), #{ desc => <<"CPU average load in 5 minute">>
+                      , example => "2.66"})}
+    , { load15
+      , mk(string(), #{ desc => <<"CPU average load in 15 minute">>
+                      , example => "2.66"})}
+    , { max_fds
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:133]: max_fds: non_neg_integer

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 4]
```diff
@@ -17,121 +17,177 @@
...
+      , mk(integer(), #{ desc => <<"File descriptors limit">>
+                       , example => 1024})}
+    , { memory_total
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:136]: emqx_schema:bytesize()

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 5]
```diff
@@ -17,121 +17,177 @@
...
+      , mk(string(), #{ desc => <<"Allocated memory">>
+                      , example => "512.00M"})}
+    , { memory_used
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:139]: emqx_schema:bytesize()

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 6]
```diff
@@ -17,121 +17,177 @@
...
+      , mk(string(), #{ desc => <<"Used memory">>
+                      , example => "256.00M"})}
+    , { node_status
+      , mk(string(), #{ desc => <<"Node status">>
+                      , example => "Running"})}
+    , { otp_release
+      , mk(string(), #{ desc => <<"Erlang/OTP version">>
+                      , example => "24.2/12.2"})}
+    , { process_available
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:148]: non_neg_integer

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 7]
```diff
@@ -17,121 +17,177 @@
...
+      , mk(integer(), #{ desc => <<"Erlang processes limit">>
+                       , example => 2097152})}
+    , { process_used
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:151]: non_neg_integer

File: apps/emqx_management/src/emqx_mgmt_api_nodes.erl [hunk 8]
```diff
@@ -17,121 +17,177 @@
...
+      , mk(integer(), #{ desc => <<"Running Erlang processes">>
+                       , example => 1024})}
+    , { uptime
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_nodes.erl:154]: non_neg_integer

**Review by zhongwencool — APPROVED** [review]

**Pull request merged**